src/applications/disability-benefits/all-claims/content/explanationOfOther.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export const explanationOfOther = (
+  <span className="hiddenTitle">Explanation of Other</span>
```

Comment:
I think there is a className already within the USDS  called `sr-only` (screenreader only). Can you try that class and see if it still works, so we're not writing extra CSS/classNames?

Reply:
@aaronjlech : It worked! Can you give it a look again? Thank you!

Reply:
lgtm!